rule/struct_tag.go

```diff
@@ -506,6 +512,44 @@ func (lintStructTagRule) typeValueMatch(t ast.Expr, val string) bool {
 
 	return typeMatches
 }
+func (w lintStructTagRule) checkPropertiesTag(t ast.Expr, options []string) (string, bool) {
+	if len(options) == 0 {
+		return "", true
+	}
+
+	hasDefault := false
+	for _, opt := range options {
+		switch {
+		case strings.HasPrefix(opt, "default"):
+			if hasDefault {
+				return "properties tag accepts only one default option", false
+			}
+			hasDefault = true
+
+			parts := strings.Split(opt, "=")
+			if len(parts) < 2 {
+				return "malformed default for properties tag", false
+			}
+
+			if !w.typeValueMatch(t, parts[1]) {
+				return "field's type and default value's type mismatch", false
+			}
+		case strings.HasPrefix(opt, "layout"):
+			parts := strings.Split(opt, "=")
+			if len(parts) < 2 || strings.TrimSpace(parts[1]) == "" {
+				return "malformed layout option for properties tag", false
+			}
+
+			if gofmt(t) != "time.Time" {
+				return "layout option is only applicable to fields of type time.Time", false
```

Comment:
Here (and at some other place) you are not mentioning it's about properties tag, while you do it on most of the errors.

So you could either:
- add it here
- consider wrapping it in the error message in the code calling this method